WBooks/TableBooks/TableBookController.swift

```diff
@@ -38,6 +39,22 @@ final class TableBookController: UITableViewController {
         tableBookView.booksTable.register(nib, forCellReuseIdentifier: CellBookView.identifier)
     }
     
+    func loadBooks() {
+        tableBookViewModel.fetchBook(onSuccess: { [weak self] in self?.reloadTable() }, onError: { [weak self] eror in self?.showError(error: eror) })
```

Comment:
Looks good, but let's divide this into multiple lines. Also, there's a typo in `eror` ("error").